chain/cosmos/chain_node.go

```diff
@@ -925,12 +925,16 @@ func (tn *ChainNode) NodeID(ctx context.Context) (string, error) {
 }
 
 // KeyBech32 retrieves the named key's address in bech32 format from the node.
-func (tn *ChainNode) KeyBech32(ctx context.Context, name string) (string, error) {
+func (tn *ChainNode) KeyBech32(ctx context.Context, name string, bech string) (string, error) {
```

Comment:
nit:

It would be helpful in the doc comment to know what values `bech` can be. If they are part of a known, finite list, an enum may be more suitable. 

Also, it would be helpful to mention a blank string for `bech` is acceptable.

Reply:
done